src/components/Login/LocalLogin.tsx

```diff
@@ -11,17 +11,15 @@ const messages = defineMessages({
   validationemailrequired: 'Not a valid email address',
   validationpasswordrequired: 'Password required',
   loginerror: 'Something went wrong when trying to sign in',
-  loggingin: 'Logging in...',
-  login: 'Login',
-  goback: 'Go back',
+  signingin: 'Signing in...',
```

Comment:
Use Unicode ellipsis:
```suggestion
  signingin: 'Signing in…',
```